### ISSUE TITLE
Water Electrolyzer Sound

### ISSUE THREAD

**Issue by MapleSight** [MachineMuse/MachineMusePowersuits#678]
I seem to have a problem with the water elctrolyzer constantly looping the sound. Maybe it's a conflict on my side?

**Comment by EzerArch** [comment]
Even outside the water? Did it start happening after a death? Put on and take the MPS helmet off once to "force" the sound to stop, and see what happens.

**Comment by MapleSight** [comment]
Yup outside of water, took the helmet off and it stopped, took the upgrade off and it stopped. And no it did not happen after death.
